### PR TITLE
Wrong radio button selection on snaplistview

### PR DESCRIPTION
**Pull request by TomaszLukysky**
Fix & example for test.

### CODE REVIEW

**Comment by TomaszLukawskiSam** [comment]
@singa2000 
I don't have the opportunity right now to check this change thoroughly but seems like the patch works.

File: src/js/profile/wearable/widget/wearable/SnapListview.js [hunk 1]
```diff
@@ -744,7 +744,9 @@
 
 				targetIndex = getIndexOfSnapListItem(targetListItem, listItems);
 
-				utilEvent.preventDefault(e);
+				if (e.target.tagName === "LABEL") { // prevents duplicate events on input inside label;
```
Comment (TomaszLukawskiSam) [inline src/js/profile/wearable/widget/wearable/SnapListview.js:747]: In here we need add also check if label contains input type radio. eg. e.target.querySelector("input[type='radio']")

Comment (singa2000) [reply src/js/profile/wearable/widget/wearable/SnapListview.js:747]: I think we need to more check, which is correct behavior.

Comment (TomaszLukawskiSam) [reply src/js/profile/wearable/widget/wearable/SnapListview.js:747]: > I think we need to more check, which is correct behavior.

Yes. I agree with you.

**Comment by TomaszLukawskiSam** [comment]
The PR is not needed.

**Pull request closed without merging**